src/txn/txn.c

```diff
@@ -308,6 +309,21 @@ __wt_txn_get_snapshot(WT_SESSION_IMPL *session)
     __txn_get_snapshot_int(session, true);
 }
 
+/*
+ * __txn_swap_snapshot --
+ *     Swap the snapshot pointers.
+ */
+static void
+__txn_swap_snapshot(uint64_t **snap_a, uint64_t **snap_b)
```

Comment:
May be declare this as a incline function and move to txn_inline

Reply:
Moved __txn_swap_snapshot to inline function in txn_inline in 87d8ff44e854eafef0bf73911ada633b4d983628 commit